pkgs/tools/security/thc-hydra/default.nix

```diff
@@ -23,6 +23,8 @@ stdenv.mkDerivation rec {
       --replace "-lcurses" "-lncurses"
   '';
 
+  makeFlags = lib.optional stdenv.isDarwin "CFLAGS='-Wno-undef-prefix'";
```

Comment:
```suggestion
  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-Wno-undef-prefix";
```

Reply:
This is exactly what I was looking for, thanks!